test/test_matmul_cuda.py

```diff
@@ -1512,13 +1522,14 @@ def test_blockwise_mxfp8_nvfp4_numerics(self, test_case_name, fast_accum, mkn, r
                 B = B.clamp(min=min_val, max=max_val)
                 B = _bfloat16_to_float4_e2m1fn_x2(B)
 
-                approx_match_sqnr_target = 15.8
+                approx_match_sqnr_target = 12.0 if torch.version.hip else 15.8
 
         C_ref = A_ref @ B_ref.t()
 
         # convert to swizzled format
-        A_scale = to_blocked(A_scale)
-        B_scale = to_blocked(B_scale)
+        if not torch.version.hip:
+            A_scale = to_blocked(A_scale)
```

Comment:
The row,major layout of the scales is exactly whats expected for rocm? OOC Is this maximal perf?

Reply:
As far as I recall, all applicable tests on ROCm 7.0 are passing. There may be additional tests that aren’t currently supported in our setup(gfx950 && ROCm 7.0).